plugins/catalog-unprocessed-entities/src/plugin.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createApiFactory,
+  createPlugin,
+  createRoutableExtension,
+  discoveryApiRef,
+} from '@backstage/core-plugin-api';
+
+import { rootRouteRef } from './routes';
+import {
+  CatalogUnprocessedEntitiesApi,
+  catalogUnprocessedEntitiesApiRef,
+} from './api';
+
+/**
+ * Plugin entry point
+ *
+ * @public
+ */
+export const catalogUnprocessedEntitiesPlugin = createPlugin({
+  id: 'catalog-unprocessed-entities',
+  routes: {
+    root: rootRouteRef,
+  },
+  apis: [
+    createApiFactory({
+      api: catalogUnprocessedEntitiesApiRef,
+      deps: { discoveryApi: discoveryApiRef },
+      factory: ({ discoveryApi }) =>
+        new CatalogUnprocessedEntitiesApi(discoveryApi),
+    }),
+  ],
+});
+
+/**
+ * Tool page for the Catalog Unprocessed Entities Plugin
+ *
+ * @public
+ */
+export const CatalogUnprocessedEntitiesPage =
```

Comment:
Could we also maybe export a hypothetical `CatalogUnprocessedEntitiesContent` that would be just the content of this page so that is could be added as a tab in the new DevTools plugin? Or if you are open to it maybe all the frontend would just live there as well? Your call but the first one would be super nice!

Reply:
I'm open to exporting more things, not sure exactly what you mean though

Reply:
Ok, so with version `1.14.0` we introduced the DevTools plugin which acts as a place to collect tooling that helps with the Ops side of Backstage - shows Backstage version, package versions, current config, etc. DevTools can be customized to add more tabs and being able to add this there would be handy, here's the docs: https://github.com/backstage/backstage/tree/master/plugins/devtools#customizing

Now, if we could break out the `<Content>` section of `UnprocessedEntities` into it's own component and then export that in the `plugin.ts` it could be used as a tab in DevTools. I think you may not even need to export it in the `plugin.ts` but just export it like any other component

Hopefully that better explains what I'm asking for?